common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -84,10 +165,14 @@ public static String getLocalHostName(int timeout) {
   }
 
   /**
+   * Check if the underlying OS is Windows.
+   */
+  public static boolean sIsWindows = SystemUtils.IS_OS_WINDOWS;
```

Comment:
I think we use `System.getProperty("os.name").startsWith("Windows");` in other places, if possible let's try to be consistent. If this way is better, let's open a ticket to use this throughout the codebase.